### ISSUE TITLE
Ability to reposition text on the icon

### ISSUE THREAD

**Issue by CDub619** [brittyazel/EnhancedRaidFrames#47]
Ability to reposition text within the icon:

https://imgur.com/a/4he7jsE

![image](https://user-images.githubusercontent.com/41877216/86501151-18883d00-bd4b-11ea-816d-18887a484d3f.png)

**Comment by brittyazel** [comment]
How is this different than #48?

**Comment by CDub619** [comment]
One is the stack text and one is the countdown of the buff duration text.

**Comment by brittyazel** [comment]
So you want me to split the text and damage into two separate fontstrings that can be positioned independently? I'll think on it and how it would play out. The icons are so small in most cases positioning the text on the button in 9 I feel would be pretty minor

**Comment by CDub619** [comment]
Yes, blizzard defaults has the stacks separate from the countdown text , I think because it’s easier to digest, having the option to move it anywhere on the icon would be even more ideal. 

I disagree on the text positioning I know a lot of ppl who set weak aura or tmw icons texts to only be in certain quadrants of the icon to allow the icon to have more visibility  , it also depends on the size and scale of the raid frames but some of the icons can be rather large. I imagine ppl using it for certain DeBuffs in raids set those to be huge too. I’ll provide an example when I log in later

**Comment by CDub619** [comment]
![image](https://user-images.githubusercontent.com/41877216/86517519-64d08d00-bdde-11ea-821d-888bfe040b93.png)

**Comment by CDub619** [comment]
![image](https://user-images.githubusercontent.com/41877216/86517530-8598e280-bdde-11ea-908b-02996932d3b7.png)
